spec/models/interaction_spec.rb

```diff
@@ -6,4 +6,22 @@
   it 'creates a valid instance of interaction' do
     expect(interaction).to be_valid
   end
+
+  describe "associations" do
+    it { should belong_to(:user) }
+  end
+  
+  describe "validations" do
+    it { should validate_length_of(:title).is_at_least(1).is_at_most(100) }
+    it { should validate_length_of(:description).is_at_least(1).is_at_most(1000) }
+  end
+
+  describe "name enum" do
```

Comment:
it { should define_enum_for(:name).with([ les attributs ici en symbol (ex: :to_create, :to_delete]) }